src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -908,10 +908,11 @@ vector<Visitor::ExpressionPtr> Visitor::visit(Parser::ArgListContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-void Visitor::visit(Parser::ExpressionListContext*) {
-  // This rule is only used by the `RelationExpression` and `BuiltInCall` rules
-  // which also are not supported and should already have thrown an exception.
-  AD_FAIL();
+std::vector<ExpressionPtr> Visitor::visit(Parser::ExpressionListContext* ctx) {
```

Comment:
Why was this not needed so far? Because we didn't have any functions yet with a variable number of arguments?

Reply:
Yes, this grammar rule is only used by a few expressions, all of which were unsupported so far (`IN`, `NOT IN`, `COALESCE`, `CONCAT`).